book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :autor
+  attr_reader :rentals
+
+  def initialize(title, autor)
+    @title = title
+    @autor = autor
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] Here again, you will need to set the "has-many" association for your book, as one book can have many rentals. Right now, this method is adding the rental to the rentals array, but you are not creating a new rental or similar to have a new Rental class, so I would suggest doing something similar to this:
```
def add_rental(person, date)
  Rental.new(date, self, person)
end
```
With this new method, you are creating a new Rental and passing the book itself to it as a parameter, so you have the "has-many" relationship set.